### PR TITLE
Add BSD 3 clause license

### PR DESCRIPTION
**Pull request by nicolasbock**
Fixes: #111

Signed-off-by: Nicolas Bock <nicolasbock@gmail.com>

### CODE REVIEW

File: COPYING [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Copyright 2020 The project authors
```
Comment (oxr463) [inline COPYING]: Did you mean for this to stay generic? If it were me, I would put "My name and contributors" or something like that.

Comment (nicolasbock) [reply COPYING]: Well, I wanted to make this sound as inclusive as possible :smile:

Comment (oxr463) [reply COPYING]: How about "Nicolas Bock and contributors"? This is your baby ;)

Comment (nicolasbock) [reply COPYING]: Alright, you convinced me :smile:

**Pull request merged**